pyproject.toml

```diff
@@ -13,10 +13,10 @@ classifiers=[
     'License :: OSI Approved :: Apache Software License',
     'Natural Language :: English',
     'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
-    'Programming Language :: Python :: 3.11',
```

Comment:
This line shouldn't be removed since supported versions are now 3.7 up to 3.11 ?
@nnansters